standards.md

```diff
@@ -0,0 +1,315 @@
+# Earth System Metrics and Diagnostics Standards
+
+## Author list
+Authors are presented in order of last name:  
+
+Chih-Chieh Chen, National Center for Atmospheric Research  
+Dani Coleman, National Center for Atmospheric Research  
+Tom Jackson, NOAA Geophysical Fluid Dynamics Laboratory   
+John Krasting, NOAA Geophysical Fluid Dynamics Laboratory  
+Yi-Hung Kuo, University of California Los Angeles  
+Jessica Liptak, NOAA Geophysical Fluid Dynamics Laboratory  
+David Neelin, University of California Los Angeles  
+Ana Ordonez, Lawrence Livermore National Laboratory   
+Aparna Radhakrishnan, Princeton University/NOAA Geophysical Fluid Dynamics Laboratory  
+Paul Ullrich, University of California Davis   
+
+
+## Table of Contents
+1. [Best Practices](#bestpractices)
+2. [DEM Framework File Format](#demframework)
+    1. [Settings file](#settingsfile)
+    2. [Contents file](#contentsfile)
+3. [Runtime Environment](#runtimeenvironment)
+4. [Common Output Bundle Format](#commonoutput)
+5. [Observational Data Format](#obsdataformat)
+6. [Observational Data Server](#obsdataserver)
+7. [Common Metric Output Format](#commonmetric)
+    1. [DIMENSIONS](#dimensions)
+    2. [RESULTS](#results)
+
+
+## Best Practices <a name="bestpractices"></a>
+
+### Nomenclature 
+In this document we refer to [MDTF](https://mdtf-diagnostics.readthedocs.io/en/latest/index.html) process-oriented diagnostics (PODs) and [CMEC](https://cmec.llnl.gov) metric kernels (MKs) as data evaluation modules (DEMs).
+
+### Supported Languages
+Python is the preferred language for development. However, there are certain cases where Python may not be performant or may not have the requisite functionality (i.e. certain statistical analysis routines). Nonetheless, if the kernel evaluation routines are not in python, python interfaces should be provided for accessing the functionality of the kernel.  
+
+If compiled code is used, a robust build system should be provided.  Compiled code should minimally rely on third-party dependencies and/or should be available through a conda package.
+
+### Python version
+Official support (fixing bugs) of python 2.7 will end on January 1, 2020.  
+
+The framework will support Python >= 3.7 and, as resources permit, version 2.7.  
+
+Funded development of new DEMs should be in Python >= 3.7.  
```

Comment:
Should this be increased?  I saw a call this week to start testing with 3.11.

Reply:
@wrongkindofdoctor said MDTF docs require Py>3.7, and CMEC has been using Py>3.6.

Reply:
For CMEC, the version of Python that the module uses doesn't have to be the same as the version that CMEC driver uses, if it's in another conda environment. So from the CMEC point of view, this version recommendation is to keep users from writing modules in deprecated Python versions.